.github/workflows/documentation-build.yaml

```diff
@@ -8,6 +9,10 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
+      - run: |
+          echo "Job triggered by ${{ github.actor }}."
+          echo "Running on a ${{ runner.os }} server hosted by GitHub."
+          echo "Branch ${{ github.ref }} of repository ${{ github.repository }}."
```

Comment:
I see that we are including this descriptive first step in all the workflows, so I was wondering if there is any mechanism provided by Github Actions to reuse part of the workflows following the DRY principle.

I've found that there are actually 3 ways of doing that, described in [this post](https://cardinalby.github.io/blog/post/github-actions/dry-reusing-code-in-github-actions/)

Could we include something like that to prevent having this 3 lines of `echo` rewritten in every workflow?

Reply:
Because none of the methods to not repeat look complex, I think we can take a step back and ask ourselves if this information is useful. I think its not, here my reasoning:

* `github.actor` is already present on the list of actions.
* `runner.os` is already printed on the very first "embedded" step called "Set up job".
* `github.ref` is already printed on the checkout step.
* `github.repository` is already also printed on the checkout step.

So maybe deleting is better than finding a way to DRY.

Reply:
Removing the step like @rh-hemartin is suggesting makes a lot of sense for me

Reply:
I am going to create a separate task to remove the duplication. Including the removal of duplicate code here will require PR testing, which I don't want to do with a push request already in progress.